README.rst

```diff
@@ -9,16 +9,16 @@ Contributing
 
 This project only requires that:
 
-#. you have `tox`_ installed
-#. you have Python 3 installed locally
+#. you have `sphinx`_ installed (`pip install sphinx`)
```

Comment:
Some people install sphinx with their system package manager and those versions of Sphinx can be absolutely ancient. We're relying on a theme that is shipped by default with 1.3.0 so it would make sense to add a lower constraint.